src/navbar/index.tsx

```diff
@@ -235,6 +145,129 @@ export function NavBar() {
   );
 }
 
+export function NavBarWebStudio() {
```

Comment:
Would prefer a more descriptive name (surely `NavBarMain` is _also_ a "webstudio navbar"?)

Reply:
Fixed